### PR TITLE
test(controller): add unit test for `config/config.go`

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### Ⅰ. Describe what this PR does
increase code coverage of `pkg/slo-controller/config/config.go`

### CODE REVIEW

File: pkg/slo-controller/config/config.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ type ColocationCfg struct {
 
 type NodeColocationCfg struct {
 	NodeSelector *metav1.LabelSelector
-	ColocationCfg
+	ColocationStrategy
```
Comment (hormes) [inline pkg/slo-controller/config/config.go:39]: Why change this api？

Comment (jasonliu747) [reply pkg/slo-controller/config/config.go:39]: Because instead of `ColocationCfg`, NodeColocationCfg shoud contains `ColocationStrategy`.

**Pull request merged**